implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -145,36 +145,48 @@ impl NodeManager {
     }
 }
 
+pub struct NodeManagerGeneralOptions<'a> {
+    pub node_name: String,
+    pub node_dir: PathBuf,
+    pub skip_defaults: bool,
+    pub enable_credential_checks: bool,
+    pub ac: Option<&'a AuthoritiesConfig>,
```

Comment:
I would move that to `NodeManagerProjectsOptions`

Reply:
Sorry, which arguments would you move? `ac` or all of them?

Reply:
`ac` only